modules/nw-ingress-controller-endpoint-publishing-strategies.adoc

```diff
@@ -11,6 +11,14 @@ The `NodePortService` endpoint publishing strategy publishes the Ingress Control
 
 In this configuration, the Ingress Controller deployment uses container networking. A `NodePortService` is created to publish the deployment. The specific node ports are dynamically allocated by {product-title}; however, to support static port allocations, your changes to the node port field of the managed `NodePortService` are preserved.
 
+.Diagram of `NodePortService`
+image::202_OpenShift_Ingress_0222_node_port.png[{product-title} Ingress NodePort endpoint publishing strategy]
+
+The preceding graphic shows the following concepts pertaining to {product-title} Ingress NodePort endpoint publishing strategy:
+
+* All the available nodes in the cluster have their own, externally accessible IP addresses. The service running in the cluster is bound to the unique NodePort for all the nodes.
+* When the client connects to a node that is down, via the `193.10.0.10` IP address in the graphic, the node port directly connects the client to an available node that is running the service and no load balancing is required. As the image shows, `193.10.0.10` is down, so another will need to be used instead.
```

Comment:
This first sentence is kind of long and I've been wondering if there's a way to make it two. Maybe like...

"When the client connects to a node that is down, for example, by way of the `193.10.0.10` IP address in the graphic, the node port directly connects the client to an available node that is running the service. In this scenario, no load balancing is required. As shown in the image, `193.10.0.10` is down and another IP address must be used instead." 

Also, we try to avoid the word 'via': https://docs.engineering.redhat.com/display/CCSENABLE/Avoid+these+words

Take my suggestion with a grain of salt--just playing with some ideas.

Reply:
Yeaaa...I'll take all the suggestions :) makes a lot of good sense to me! Definitely can see needing to end that first sentence and change up the future tense/via.